lib/jsdom/living/nodes/HTMLTableElement-impl.js

```diff
@@ -16,10 +18,60 @@ class HTMLTableElementImpl extends HTMLElementImpl {
     return firstChildWithHTMLLocalName(this, "thead");
   }
 
+  set tHead(value) {
+    if (value !== null && value._localName !== 'thead') {
```

Comment:
We should be checking the namespace here (and for tfoot) in theory.

I imagine there's a to-upstream WPT waiting to be written there, although it's probably not worth the trouble...

Reply:
I think this should already be managed by idl brand checks since it's a htmlsectionelement?

Reply:
Oh, fascinating! I think you're right.